src/app/app.component.ts

```diff
@@ -245,6 +240,22 @@ export class ArlasWuiComponent implements OnInit, AfterViewInit {
       if (this.configService.getValue('arlas.web.options.indicators')) {
         this.showIndicators = true;
       }
+
+      /** init from url */
+      const queryParamVisibleVisualisations = this.getParamValue('vs');
+      if (queryParamVisibleVisualisations) {
+        const visibleVisuSet = new Set(queryParamVisibleVisualisations.split(';').map(n => decodeURI(n)));
+        this.mapComponentConfig.visualisations_sets.forEach(v => v.enabled = visibleVisuSet.has(v.name));
+      }
+
+      const analyticOpenString = this.getParamValue('ao');
+      if (analyticOpenString) {
```

Comment:
You shoud use this notation `if( !!analyticOpenString) {`

otherwise there is no point to use `(analyticOpenString === 'true')` after

Reply:
fixed